sqlitecluster/SQLiteNode.cpp

```diff
@@ -1818,16 +1733,6 @@ void SQLiteNode::_onConnect(SQLitePeer* peer) {
     login["Version"] = _version;
     login["Permafollower"] = _originalPriority ? "false" : "true";
     _sendToPeer(peer, login);
-
-    // If we're STANDINGUP when a peer connects, send them a STATE message so they know they need to APPROVE or DENY the standup.
-    // Otherwise we will wait for their response that's not coming,and can eventually time out the standup.
-    if (_state == SQLiteNodeState::STANDINGUP) {
-        SData state("STATE");
-        state["StateChangeCount"] = to_string(_stateChangeCount);
-        state["State"] = stateName(_state);
-        state["Priority"] = SToStr(_priority);
-        _sendToPeer(peer, state);
-    }
```

Comment:
Since we're not using any locks here, would it be possible that we have a state while sending login and another when sending state?

And in that case, considering that the _changeState would have been called somewhere else, would everything still work out fine since _changeState also sends a STATE message?

Reply:
It would have been a bug if state changed between these two messages, though that couldn't happen any more since this change as we only send one now, right?

But state couldn't change between these two messages as only the sync thread can change state, and only the sync thread calls `_onConnect`.

Does that adequately answer your question?

Reply:
It does, thanks!